test/test_humanid.py

```diff
@@ -0,0 +1,41 @@
+import json
+import unittest
+
+import responses
+
+import base
+import requests
+
+class HumanIdTester(unittest.TestCase):
+    def setUp(self):
+        self._plugin = base.load_plugin('humanid.py', 'HumanId')
+        self._proto = base.TestProto([self._plugin])
+
+    @base.net_test
+    def test_query_real(self):
+        m = self._plugin.get_some()
+        self.assertEqual('A process for testing things', m)
```

Comment:
This'll fail (every?) time.  Best you can probably do here is just make sure that ```m``` has non-zero length.